### PR TITLE
Escape commit message when doing `hg amend`

### PR DESCRIPTION
**Pull request by elthariel**
We're running a command with an arbitrary string unescaped, and it sometimes breaks. That's also a security issue

### CODE REVIEW

File: lib/between_meals/repo/hg/cmd.rb [hunk 1]
```diff
@@ -44,7 +46,7 @@ def username
         end
 
         def amend(msg)
-          cmd("commit --amend -m '#{msg}'")
+          cmd("commit --amend -m '#{Shellwords.escape msg}'")
```
Comment (jaymzh) [inline lib/between_meals/repo/hg/cmd.rb]: style: parenthesis around function args, please

**Comment by jaymzh** [comment]
:+1:

File: lib/between_meals/repo/hg/cmd.rb (hunk 1 above)
Comment (odcinek) [inline lib/between_meals/repo/hg/cmd.rb]: lint is in travis, will catch it

File: lib/between_meals/repo/hg/cmd.rb [hunk 2]
```diff
@@ -44,7 +46,14 @@ def username
         end
 
         def amend(msg)
-          cmd("commit --amend -m '#{msg}'")
+          begin
+            f = Tempfile.new('beetwen_meals.hg.amend')
```
Comment (jaymzh) [inline lib/between_meals/repo/hg/cmd.rb]: beetwen_meals? I think you mean between_meals :)

**Comment by jaymzh** [comment]
@elthariel rebase and push to your branch and you'll use the new travis stuff that should actually run the damn tests.

**Comment by jaymzh** [comment]
er... you merged the rest of the world on you... so I dunno what this will do, but sure.

**Comment by elthariel** [comment]
Yeah, forgot to rebase like a n00b. Damn mercurial i lose my good habits

**Pull request merged**